src/_data/sites/ja.yml

```
@@ -0,0 +1,542 @@
+      新機能をレビューし、実装する人々
+  committers:
+    title: コミッター
+    description: >
+      バグをレビューして修正し、問題のトリアージを支援する人々
+  alumni:
+    title: アルムナイ
+    description: >
+      以前、ESLintの保守を手伝っていた元チームメンバー
+
+#------------------------------------------------------------------------------
+# Sponsors Page
+#------------------------------------------------------------------------------
+
+sponsors_page:
+  title: スポンサー様
+  description: >
+    現在、BACKER_COUNT の企業、団体、個人が
+    ESLintの継続的な維持・発展を支援するために、毎月総額MONTHLY_DONATIONSの寄付をしています。
+  hero:
+    cta:
+      text: スポンサーになる
+      url: /donate
+  tiers:
+    platinum:
+      title: プラチナスポンサー
+      description: >
+        毎月2000ドルの寄付で、プラチナスポンサーとなり、40時間のメンテナンスと開発に貢献します。
+        2時間の専属サポートサービスをもらえます。サポートはチャットや画面共有による遠隔サポートの形でできます。
+        プラチナスポンサーのアバター画像はESLintのREADMEとeslint.orgのトップページに記載いたします。
+        また フォロワー数2万人以上のESLintのTwitterアカウントからお礼をツイートします。
+        サポートセッションをご希望の方は、ESLintチームまでご連絡ください。
+    gold:
+      title: ゴールドスポンサー
+      description: >
+        毎月1000ドルの寄付で、ゴールドスポンサーとなり、20時間のメンテナンスと開発に貢献します。
+        ゴールドスポンサーのアバター画像はESLintのREADMEとeslint.orgのトップページに記載いたします。
+        また フォロワー数2万人以上のESLintのTwitterアカウントからお礼をツイートします。
+    silver:
+      title: シルバースポンサー
+      description: >
+        毎月500ドルの寄付で、シルバースポンサーとなり、10時間のメンテナンスと開発に貢献します。
+        シルバースポンサーのアバター画像はESLintのREADMEとeslint.orgのトップページに記載いたします。
+    bronze:
+      title: ブロンズスポンサー
+      description: >
+        毎月200ドルの寄付で、ブロンズスポンサーとなり、4時間のメンテナンスと開発に貢献します。
+        ブロンズスポンサーのアバター画像はESLintのREADMEとeslint.orgのトップページに記載いたします。
+    technology:
+      title: テクノロジースポンサー
+      description: >
+        技術スポンサーはオープンソースのエコシステムと私たちの仕事への貢献の一環として、製品・サービスを無償で使わせていただいています。
+    backers:
+      title: バッカー
+      description: >
+        バッカー（支援者）は、ESLintを維持するための資金を毎月提供します。
+
+  recent_donations:
+    title: 最近の一時的な寄付
+    description: >
+      また、素晴らしい方々から一時的な寄付もたくさんいただいています。
+      OpenCollectiveとGitHubから最近の寄付を紹介します。
+
+  sponsor_title: SPONSOR_NAME は毎月 AMOUNTの寄付感謝いたします。
+  tech_sponsor_title: TECH provided by SPONSOR_NAME
+
+#------------------------------------------------------------------------------
+# Donate Page
+#------------------------------------------------------------------------------
+
+donate_page:
+  title: Donate
+  description: >
+    ESLintは、継続的なメンテナンスと開発のための寄付金に依存しています
+    現在、BACKER_COUNTの企業・団体・個人が、毎月MONTHLY_DONATIONSを寄付してくれています。
+  hero:
+    cta:
+      text: スポンサーを見る
+      url: /sponsors
+  donation_options:
+    title: 寄付方法
+    subtitle: 以下のサイトを通じて寄付を受け付けています。
+    open_collective:
+      title: Open Collective
+      subtitle: 法人向けで一番人気のプラン
+      cta:
+        text: Open Collectiveを通じて寄付をします。
+      items:
+        - text: 501(c)(3)非営利団体に直接寄付をする
+        - text: 一時的な寄付と定期的な寄付
+        - text: クレジットカード、銀行振り込み、PayPalで支払います
+        - text: オープンソースファンドを設立し、複数のプロジェクトに寄付をします
+        - text: ほとんどの国で利用できます
+    github_sponsors:
+      title: GitHubを通じて寄付をします
+      subtitle: 高度な機能とレポート機能
+      cta:
+        text: Githubで寄付をする
```

I think it's better.

```suggestion
        text: GitHubで寄付をする
```

---

Since this is a call to action, a noun format might be better:
```suggestion
        text: GitHubを通じた寄付
```